src/test/java/uk/gov/digital/ho/hocs/workflow/security/AllocatedAspectTest.java

```diff
@@ -25,6 +26,9 @@ public class AllocatedAspectTest {
     @Mock(answer = Answers.RETURNS_DEEP_STUBS)
     private CaseworkClient caseworkClient;
 
+    @Mock(answer = Answers.RETURNS_DEEP_STUBS)
```

Comment:
Does this need `(answer = Answers.RETURNS_DEEP_STUBS)`?

Reply:
👻 whooo knowws, i'll take it out